File: c++/Elektronik.cpp
#include <string>
#include <iostream>
using namespace std;

class Elektronik {
protected:
    string merk;
    int tahunProduksi;
    double harga;

public:
    Elektronik(string merk, int tahunProduksi, double harga);
    string getMerk();
    void setMerk(string merk);
    int getTahunProduksi();
    void setTahunProduksi(int tahunProduksi);
    double getHarga();
    void setHarga(double harga);
    virtual void displayInfo();
};

string Elektronik::getMerk() { return merk; }
void Elektronik::setMerk(string merk) { this->merk = merk; }

int Elektronik::getTahunProduksi() { return tahunProduksi; }
void Elektronik::setTahunProduksi(int tahunProduksi) { this->tahunProduksi = tahunProduksi; }

double Elektronik::getHarga() { return harga; }
void Elektronik::setHarga(double harga) { this->harga = harga; }

void Elektronik::displayInfo() {
    cout << "Merk: " << merk << endl;
    cout << "Tahun Produksi: " << tahunProduksi << endl;
    cout << "Harga: " << harga << endl;
}

File: java/Elektronik.java
public class Elektronik {
    private String merk;
    private int tahunProduksi;
    private double harga;

    public Elektronik(String merk, int tahunProduksi, double harga) {
        this.merk = merk;
        this.tahunProduksi = tahunProduksi;
        this.harga = harga;
    }

    public String getMerk() { return merk; }
    public void setMerk(String merk) { this.merk = merk; }

    public int getTahunProduksi() { return tahunProduksi; }
    public void setTahunProduksi(int tahunProduksi) { this.tahunProduksi = tahunProduksi; }

    public double getHarga() { return harga; }
    public void setHarga(double harga) { this.harga = harga; }

    public void displayInfo() {
        System.out.println("Merk: " + merk);
        System.out.println("Tahun Produksi: " + tahunProduksi);
        System.out.println("Harga: " + harga);
    }
}